src/ansys_sphinx_theme/theme/ansys_sphinx_theme/static/js/search.js

```diff
@@ -28,12 +28,13 @@ require(["fuse"], function (Fuse) {
   // Initialize Fuse when the data is fetched
   function initializeFuse(data, options) {
     fuse = new Fuse(data, options);
+    // add env variable "FUSE_ACTIVE" to indicate that the search is ready
+    document.documentElement.setAttribute("data-fuse_active", "true");
```

Comment:
https://github.com/ansys/ansys-sphinx-theme/pull/584/files#r1838131150 adding the value only if fuse.js initialize